flow/templates/custom_script_template.sh

```diff
@@ -0,0 +1,15 @@
+{#
+This is a jinja template used to generate another jinja template.
+This file is used by `flow template create` to detect the current environment
+and create a new script template that can be modified by the user to add
+custom commands (e.g. `module load ...`).
+#}
+{{ '{% extends "' + extend_template + '" %}' }}
+{% raw %}
+{% block header %}
+    {{- super () -}}
+{% endblock header %}
```

Comment:
I wonder if there are other blocks that we should add in this template, with recommendations for usage? Like maybe `preamble` or `project_header`? Not sure where the best place is for a `module load` command, but `module load` is the specific use case I would try to simplify.

Maybe we could add a comment block `{# ... #}` that is in the "raw" section and give instructions for how to use the template effectively.

Reply:
I thought of this, but didn't know what advice to exactly offer, so punted it for now. If we think this needs to be there to start, I can try my hand at adding a message.

Reply:
I created #545 to track this. I think it would increase the value of the feature but we don't have to solve it right now.